src/hotspot/share/services/memTracker.cpp

```diff
@@ -183,6 +183,9 @@ void MemTracker::final_report(outputStream* output) {
     if (level >= NMT_summary) {
       report(level == NMT_summary, output);
     }
+#ifdef ASSERT
+    tuning_statistics(output);
```

Comment:
This is irrelevant.  tuning_statistics() is undocumented and should not advertise (at least, it was the decision then).

Otherwise, looks good.

Reply:
Okay, I remove it before pushing. Thank you.

Reply:
Thanks.